lib/rspec/sidecar/services.rb

```diff
@@ -15,9 +15,14 @@ def service(name, type,
               zookeeper_port = ENV["ZOOKEEPER_PORT_2181_TCP_PORT"] || 2181)
     ZK.open(zookeeper_host + ":" + zookeeper_port.to_s) do |zk|
       instances_path = "/banno/services/#{name}:#{type}"
-      instances = zk.children(instances_path)
-      instance_json = zk.get(instances_path + "/" + instances[0])[0]
-      JSON.parse(instance_json)
+      loop do
+        instances = zk.children(instances_path)
+        unless instances.empty?
+          instance_json = zk.get(instances_path + "/" + instances[0])[0]
+          return JSON.parse(instance_json)
+        end
+        sleep 1
+      end
```

Comment:
What happens if an image doesn't work right and no service ever registers itself with zookeeper?  Won't this just spin forever and if it's in a jenkins build someone will just have to notice?